src/Components/WebAssembly/Authentication.Msal/src/Interop/AuthenticationService.ts

```diff
@@ -251,9 +254,7 @@ class MsalAuthorizeService implements AuthorizeService {
 
     async completeSignOut(url: string) {
         const logoutStateId = sessionStorage.getItem(`${AuthenticationService._infrastructureKey}.LogoutState`);
-        const updatedUrl = new URL(url);
-        updatedUrl.search = `?state=${logoutStateId}`;
-        const logoutState = await this.retrieveState(updatedUrl.href, /*isLogout*/ true);
+        const logoutState = this.retrieveState(logoutStateId, /*isLogout*/ true);
```

Comment:
@javiercn What's the reasoning for propagating the logout state key over the URL instead of passing it directly as a parameter here?

It was particularly buggy when I tested it out (SearchParams were not populated correctly with state property) and didn't seem required hence the change here.

Reply:
@captainsafia state must not be guessable.

Reply:
Thanks! I've updated this to continue using the state query parameter and read from it directly in `retrieveState` instead of using the hash.